mrp_subcontracting_skip_no_negative/models/stock_move.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _action_done(self, cancel_backorder=False):
+        moves_with_no_check = self.filtered(lambda x: x.is_subcontract).with_context(
+            skip_negative_qty_check=True
+        )
+        # For rather unlikely occassions where linked production is not in the right
+        # state.
+        for move in moves_with_no_check:
+            production_move = self.search([("move_dest_ids", "=", move.id)])
+            production_mo = production_move.production_id
```

Comment:
```suggestion
            production = production_move.production_id
```
Sorry to be picky. XD